src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,53 +1,126 @@
+// import './NewMovie.scss';
+// import classNames from 'classnames';
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (newMovie: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+
+  const [title, setTitle] = useState('');
+
+  const [description, setDescription] = useState('');
+
+  const [imdbId, setImdbId] = useState('');
+
+  const [imgUrl, setImgUrl] = useState('');
+
+  const [imdbUrl, setImdbUrl] = useState('');
+
+  let isDisabled = true;
+
+  if (title && imgUrl && imdbUrl && imdbId) {
+    isDisabled = false;
+  }
```

Comment:
```suggestion
  let isDisabled = !title || !imgUrl || !imdbUrl || !imdbId;
```
add `.trim()` to each field here